src/test/java/org/broadinstitute/barclay/help/testinputs/TestArgumentContainer.java

```diff
@@ -211,6 +213,11 @@ public String getHelpDoc() {
             optional = true)
     public List<File> mutexTargetField2;
 
+    @Argument(shortName = "ES", fullName = "enumSetLong",
```

Comment:
I can't see if there's one in here, but if not can you add in a `List<List<String>>` argument?

Reply:
Same comment as above; thats kind of thing isn't supported by the arg parser.

Reply:
Sounds good.